src/components/IndexHeader/index.js

```diff
@@ -29,7 +29,7 @@ export default class IndexHeader extends Component {
         <div className="container">
           <div className="row">
             <div className="col-md-10 col-md-offset-1 text-center">
-              <Link to="/" data-metrics-event-name="IndexHeader:Link:Index">
+              <Link to="/" className="link" data-metrics-event-name="IndexHeader:Link:Index">
```

Comment:
minor suggestion: we use css-modules via webpack, this allows us to require style styles, and it also means less globals in our css. 

```js

const styles = require('./style.scss')]

<Link to="/" className={styles.link} data-metrics-event-name="IndexHeader:Link:Index">
```

If you have time, this is a preferred approach. 
Example: https://github.com/quran/quran.com-frontend/blob/master/src/components/Ayah/index.js#L13

docs:
https://github.com/css-modules/css-modules